src/main/scala/nl.knaw.dans.easy.solr4files/package.scala

```diff
@@ -16,49 +16,57 @@
 package nl.knaw.dans.easy
 
 import java.io.File
-import java.net.{URL, URLDecoder}
+import java.net.{ URL, URLDecoder }
 
-import nl.knaw.dans.lib.error.{CompositeException, _}
+import nl.knaw.dans.lib.error.{ CompositeException, _ }
 import nl.knaw.dans.lib.logging.DebugEnhancedLogging
 import org.apache.commons.io.FileUtils.readFileToString
 import org.apache.solr.common.util.NamedList
 
-import scala.util.{Failure, Success, Try}
-import scala.xml.{Elem, XML}
-import scalaj.http.{Http, HttpResponse}
+import scala.collection.mutable
+import scala.util.{ Failure, Success, Try }
+import scala.xml.{ Elem, XML }
+import scalaj.http.{ Http, HttpResponse }
 
 package object solr4files extends DebugEnhancedLogging {
 
   type FeedBackMessage = String
   type SolrLiterals = Seq[(String, String)]
   type FileToShaMap = Map[String, String]
 
-  case class WrappedCompositeException(msg: String, cause: CompositeException )
-    extends Exception(s"$msg ${cause.getMessage()}", cause)
+  case class WrappedCompositeException(msg: String, cause: CompositeException)
```

Comment:
I'm not yet a fan of this name. Maybe you could think of a better name or have better namings for the specific use cases.